segments/gitstatus/gitstatus.p9k

```diff
@@ -0,0 +1,146 @@
+    if [[ ${P9K_MODE} != 'compatible' ]]; then
+      #                                                                                                                         
+      p9k::register_icon "GITSTATUS_BRANCH"        $'\uE0A0 '  $'\uE220 '  $'\uF126 '  '\u'${CODEPOINT_OF_OCTICONS_GIT_BRANCH}' '        $'\uF126 '
+    else
+      #                                                                                                                         
+      p9k::register_icon "GITSTATUS_BRANCH"        "@ "        $'\uE220 '  $'\uF126 '  '\u'${CODEPOINT_OF_OCTICONS_GIT_BRANCH}' '        $'\uF126 '
+    fi
+  fi
+
+  #                                          →           →          →                                                            
+  p9k::register_icon "GITSTATUS_REMOTE_BRANCH"     $'\u2192'   $'\u2192'   $'\u2192'   '\u'${CODEPOINT_OF_OCTICONS_REPO_PUSH}            $'\uE728 '
+  #                                                                                                                             
+  p9k::register_icon "GITSTATUS_GIT"               ''          $'\uE20E '  $'\uF1D3 '  '\u'${CODEPOINT_OF_AWESOME_GIT}' '                $'\uF1D3 '
+  #                                                                                                                              
+  p9k::register_icon "GITSTATUS_GITHUB"        ''          $'\uE20E '  $'\uF113 '  '\u'${CODEPOINT_OF_AWESOME_GITHUB_ALT}' '         $'\uF113 '
+  #                                                                                                                              
+  p9k::register_icon "GITSTATUS_BITBUCKET"     ''          $'\uE20E '  $'\uF171 '  '\u'${CODEPOINT_OF_AWESOME_BITBUCKET}' '          $'\uE703 '
+  #                                                                                                                              
+  p9k::register_icon "GITSTATUS_GITLAB"        ''          $'\uE20E '  $'\uF296 '  '\u'${CODEPOINT_OF_AWESOME_GITLAB}' '             $'\uF296 '
+
+  ################################################################
+  # Define the actual segment
+  ################################################################
+
+  ################################################################
+  # Register segment
+  # Parameters:
+  #   segment_name  context  background  foreground  Generic  Flat/Awesome-Patched  Awesome-FontConfig  Awesome-Mapped-FontConfig  NerdFont
+  p9k::register_segment  "GITSTATUS" "CLEAN"     "green"  "${DEFAULT_COLOR}" ''  ''  ''  ''  ''
+  p9k::register_segment  "GITSTATUS" "MODIFIED"  "yellow" "${DEFAULT_COLOR}" ''  ''  ''  ''  ''
+  p9k::register_segment  "GITSTATUS" "UNTRACKED" "green"  "${DEFAULT_COLOR}" '?' $'\uE16C' $'\uF059' '\u'${CODEPOINT_OF_AWESOME_QUESTION_CIRCLE} $'\uF059'
+
+  ################################################################
+  # Register segment default values
+  p9k::set_default P9K_GITSTATUS_MAX_INDEX_SIZE_DIRTY "4096"
```

Comment:
-1 would be a better default. The latest version if gitstatus (not submitted yet) returns data for chromium (300k files) in less than 75 ms. This is 5 times faster than `git status`. It'll feel fast even without async.

Reply:
Awesome! I'll change it to `-1`, as you are way faster than me anyway. ;)